envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/groups/NodeMetadataValidator.kt

```diff
@@ -21,6 +28,8 @@ class ConfigurationModeNotSupportedException(serviceName: String?, mode: String)
 class NodeMetadataValidator(
     val properties: SnapshotProperties
 ) : DiscoveryServerCallbacks {
+    private val logger by logger()
```

Comment:
I think we don't need a logger. We don't log any other validation exceptions here, because the exceptions are logged anyway (they are also logged in envoy of a service that sent invalid config, so people will see it in their service logs with ERROR level)

Reply:
I missed that we log an exception here, so logging makes sense. My mistake.